x-pack/plugins/lens/public/pie_visualization/partition_charts_meta.ts

```diff
@@ -37,6 +38,11 @@ interface PartitionChartMeta {
       value: SharedPieLayerState['numberDisplay'];
       inputDisplay: string;
     }>;
+    donutInnerAreaSizeOptions: Array<{
```

Comment:
metadata should keep a generic data. let's exclude everything with a special chart name